client/components/Description.jsx

```diff
@@ -1,18 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Description = ({ descriptionBody }) => (
-  <div className="Description">
-    {descriptionBody}
+const Description = ({ descriptionBody, toggle, showDescription }) => (
+  <div className="description">
+
+    <div className="description-top">
+      {descriptionBody.split('.').slice(0, 5).join('.')}
+    </div>
+
+    <div className={`description-bottom-${showDescription}`}>
+      {descriptionBody.split('\n').map(
+        par => (
+          <p>
+            {par}
+          </p>
+        ),
+      )}
+    </div>
+
+    <span role="button" tabIndex="0" className="read-more" onClick={() => { toggle(); }} onKeyPress={() => { toggle(); }}>
```

Comment:
what is the purpose of onKeyPress on <span> tag?

Reply:
For a non-button clickable element, eslint solicites a keyboard interaction option for accessibility purposes.